conf/vsc_ugent.config

```diff
@@ -42,7 +33,25 @@ process {
     stageInMode = "symlink"
     stageOutMode = "rsync"
     errorStrategy = { sleep(Math.pow(2, task.attempt ?: 1) * 200 as long); return 'retry' }
-    maxRetries    = 5
+    // add GPU support with GPU label 
+    // Adapted from https://github.com/nf-core/configs/blob/76970da5d4d7eadd8354ef5c5af2758ce187d6bc/conf/leicester.config#L26
+    // More info on GPU SLURM options: https://hpc.vub.be/docs/job-submission/gpu-job-types/#gpu-job-types
+    withLabel: use_gpu {
+        // works on all GPU clusters of Tier 1 and Tier 2
+        beforeScript = 'module load cuDNN/8.4.1.50-CUDA-11.7.0'
+        // TODO: Support multi-GPU configurations with e.g. ${task.ext.gpus}
+        clusterOptions = {"--gpus=1 --account=$tier1_project"}
```

Comment:
Can you only add the account flag here if the `tier1_project` variable is set? I think this will cause issues for tier 2 clusters otherwise